### PR TITLE
API-125: Move the JavaScript SDK publishing to github actions

### PR DESCRIPTION
**Pull request by Arpit-Sharma-USC**
[JIRA](https://bynder.atlassian.net/browse/API-125)

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm run build --if-present
```
Comment (githuib) [inline .github/workflows/run-tests.yml:31]: Just wondering, what is the `--if-present` needed for here? Shouldn't it always have a `build` task?

Comment (Arpit-Sharma-USC) [reply .github/workflows/run-tests.yml:28]: This is something the boiler plate generates automatically, but yes you are right the build task is present by default making this extraneous 😄

File: package.json [hunk 2]
```diff
@@ -45,5 +45,8 @@
     "query-string": "^6.1.0",
     "simple-oauth2": "^2.5.1"
   },
+  "resolutions": {
+    "graceful-fs": "^4.2.4"
```
Comment (betacar) [inline package.json:49]: What's this for?

**Review by betacar — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Arpit-Sharma-USC) [reply package.json:49]: This fixes a Reference error primordials is not defined which occurs when a dependency under gulp does that monkeypatch node.js.

This overrides and performs that monkeypatch 😄

**Pull request merged**